### PR TITLE
updated links to STS

### PR DESCRIPTION
**Pull request by kumuduh**
Updated Links to STS instead of IAM

### CODE REVIEW

File: content/docs/rosa/prereq-list/index.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Once you have the above prerequisites installed and enabled, let's proceed to th
 
 
 ## SCP Prerequisites
-Ensure that your organization's [service control policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) has the minimum set of effective permissions as detailed [here](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.html#rosa-minimum-scp_prerequisites).
+Ensure that your organization's [service control policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) has the minimum set of effective permissions as detailed [here](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshoot-rosa-enablement.html#error-aws-orgs-scp-denies-permissions).
```
Comment (dsari-rh) [inline content/docs/rosa/prereq-list/index.md:62]: should we instead add a bullet point underneath something along the line "Ensure that your SCP is configured to allow the required `aws-marketplace:Subscribe` permission when you choose `Enable ROSA` from the console, and please refer [here](https://docs.aws.amazon.com/ROSA/latest/userguide/troubleshoot-rosa-enablement.html#error-aws-orgs-scp-denies-permissions) for more details." ?

Comment (kumuduh) [reply content/docs/rosa/prereq-list/index.md:62]: yes that's a good idea.

Comment (supernovae) [reply content/docs/rosa/prereq-list/index.md:62]: @kumuduh can you get the language updated and add me as a reviewer so we can get this merged in? ty!

**Comment by michaelryanmcneill** [comment]
Issue has gone stale, closing.

**Pull request closed without merging**